README.md

```diff
@@ -786,34 +770,54 @@ This indicates that there were two files uploaded via the `license_upload` widge
 are `34` and `47`, and can be looked up in the `user_files` table (detailed below) using those ids.
 
 The `user_files` table holds information about a submission's uploaded user files, which includes:
-`file_id`, the corresponding submission's `submission_id`, a `created_at` time,
-the `original_name` of the file, and the S3 `repository_path`.
+`file_id`, the corresponding submission's `submission_id`, a `created_at` time, the `original_name`
+of the file, and the S3 `repository_path`.
 
 ### Deleting Uploaded Files
 
 Upon successful upload a link is provided to delete the file. When the link is used, a browser
 native delete confirmation pop-up will appear asking the user if they are sure they want to delete
-the selected file. If the user selects yes from the pop-up, the file will be deleted
-from the `user_files` table in the database, as well as from S3 storage.
+the selected file. If the user selects yes from the pop-up, the file will be deleted from
+the `user_files` table in the database, as well as from S3 storage.
 
 ### S3 File Retention Policies
 
 We recommend setting a maximum file retention period in S3 by setting up a retention policy on your
-bucket.
-This will automatically delete files in your bucket that are older than the retention policy
+bucket. This will automatically delete files in your bucket that are older than the retention policy
 permits.
 [You can read more about configuring a retention policy in S3 here.](https://docs.aws.amazon.com/AmazonS3/latest/userguide/how-to-set-lifecycle-configuration-intro.html)
 
+### Address Validation
+
+`Form-flow` will support address validation through [Smarty](https://www.smarty.com/). Configuration
+information for smarty can be found [here].
```

Comment:
`[here]` seems to need a link still - do you have a place in mind or is this still a TODO?

Reply:
It was a potential link to smarty section of our code, where we would outline configuration information for the smarty install.